rebound/particle.py

```diff
@@ -135,6 +136,11 @@ def __init__(self, particle=None, m=None, x=None, y=None, z=None, vx=None, vy=No
             self.vy = vy
             self.vz = vz
 
+        if ID is None:
+            self.ID = rebound.module.lastID
```

Comment:
I don't think it matters, but there might be a confusion given that `lastID` is really `lastID+1`.